apps/web/src/pages/request-support/index.tsx

```diff
@@ -19,8 +19,8 @@ export default function RequestSupport({ returnPath }: RequestSupportProps) {
           <h1 className="govuk-heading-l">Request NIHR RDN support</h1>
           <p>
             Contact{' '}
-            <Link href="https://www.nihr.ac.uk/documents/study-support-service-contacts/11921">your local network</Link> if
-            you would like to discuss how the NIHR RDN may be able to support you to deliver your study.
+            <Link href="https://www.nihr.ac.uk/documents/study-support-service-contacts/11921">your local network</Link>{' '}
```

Comment:
What's the `{' '}`?

Reply:
Enforces a space in jsx, it's been added by the lint/prettier rules automatically but this is the right way to do it so was left in